src/components/utils/_mixins.scss

```diff
@@ -2,4 +2,7 @@
   background-color: $theme;
   box-shadow: 0 0 1px rgba($theme, 0.25);
   color: $white;
+  transition:
+    background-color 2s,
+    color 2s;
```

Comment:
I think we already discussed this before - you should mention the properties first and then set the timing once.

And 2 seconds maybe is too much.

Reply:
i made it but it doesnt work, if i use    background-color,
    color 2s; only color has 2s